packages/store/src/internal/state-operations.ts

```diff
@@ -12,11 +12,19 @@ import { deepFreeze } from '../utils/freeze';
  */
 @Injectable()
 export class InternalStateOperations {
-  constructor(
-    private _stateStream: StateStream,
-    private _dispatcher: InternalDispatcher,
-    private _config: NgxsConfig
-  ) {}
+  constructor(private _stateStream: StateStream, private _dispatcher: InternalDispatcher, private _config: NgxsConfig) {
```

Comment:
Please try to avoid reformatting code unrelated to the change. I suspect that you may have an automatic tool that is applying its own style?